src/NetTopologySuite/Geometries/Implementation/PackedCoordinateSequence.cs

```diff
@@ -46,6 +46,32 @@ public sealed override Coordinate GetCoordinate(int i)
             return GetCoordinateCopy(i);
         }
 
+        /// <summary>
+        /// Returns whether the i'th coordinate is valid
+        /// </summary>
+        public virtual bool IsValidCoordinate(int i)
+        {
+            double d = GetX(i);
+            if (double.IsInfinity(d) || double.IsNaN(d)) return false;
```

Comment:
`if (!double.IsFinite(d)) return false;`

Reply:
It is not netstandard2.0.. .it does not compile.